task22/js/main.js

```diff
@@ -68,13 +68,17 @@ const getCreatedTableHeader = (userTableColumn) => {
     const theadElement = document.createElement("thead");
     const tableRowElement = document.createElement("tr");
     const tableHeaderFragment = document.createDocumentFragment();
-    for(const column of Object.values(userTableColumn)){
+    const sortTarget = ["id", "age"];
```

Comment:
FYI
When I worked on the same issue before, Monaka-san told me that there is a way to manage sorting elements with such an object. I am posting it here for your reference.
https://github.com/nakagawamai/js-lesson/pull/24#discussion_r967828102

```JavaScript
const userTableColumn = {
  id: {
    value: "ID",
    hasSort: true
  },
  name: {
    value: "名前",
    hasSort: false
  },
  gender: {
    value: "性別",
    hasSort: true
  },
  age: {
    value: "年齢",
    hasSort: false
  }
};
```

Reply:
Thanks for the helpful advice. I will refer to it the next time I implement it. It was a great learning experience because it was an idea that was too new to me.
I will leave it as is for this implementation.